.github/workflows/stale.yml

```diff
@@ -0,0 +1,17 @@
+name: 'Close stale PRs'
+on:
+  schedule:
+    - cron: '30 1 * * *'
+
+jobs:
+  stale:
+  runs-on: ubuntu-latest
+  steps:
+  - name: Close Stale PRs
+    uses: actions/stale@v5.0.0
+    with:
+      remove-pr-stale-when-updated: true
+      days-before-issue-close: -1
```

Comment:
why not close issues?

Reply:
We have some longstanding issue and would be tedious to reopen them every now and than

Reply:
@iTitus how do we proceed here? Let's start with PR staling and leave the issues for now?